frontend/src/v5/ui/routes/userSignup/userSignupForm/userSignupForm.styles.ts

```diff
@@ -0,0 +1,73 @@
+/**
+ *  Copyright (C) 2022 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+import styled from 'styled-components';
+import { Link } from 'react-router-dom';
+import StepperBase from '@mui/material/Stepper';
+import StepLabelBase from '@mui/material/StepLabel';
+import { Content } from '../userSignup.styles';
+
+export const Container = styled.div`
+	flex: 1;
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	justify-content: center;
+`;
+
+export const Title = styled.div`
+	${({ theme }) => theme.typography.h1};
+	color: ${({ theme }) => theme.palette.secondary.main};
+`;
+
+export const Underlined = styled.div`
+	display: inline-block;
+	text-decoration: underline;
+	text-underline-offset: 3px;
+`;
+
+export const Stepper = styled(StepperBase)`
+	${Content}
```

Comment:
The `${Content}` is a piece of code produce with the `css\`\`` function and used to style the box around the registration stepper and the confirmation message. Since the style is the same for both, to avoid code duplication, the code was re-used. I refactored it to make it cleared